Helper/Entity/Product/PriceManager/ProductWithoutChildren.php

```diff
@@ -325,9 +374,10 @@ protected function addTierPrices($tierPrice, $field, $currencyCode)
     /**
      * @param $groupId
      * @param $product
+     * @param $subProducts
      * @return float
      */
-    protected function getRulePrice($groupId, $product)
+    protected function getRulePrice($groupId, $product, $subProducts)
```

Comment:
I am not too nuts about this modification to the API because the parent class never utilizes the `$subProducts` parameter. This violates the Liskov Substitution Principal - namely that objects of a superclass should be replaceable by objects of a subclass but here we are modifying the interface. 

I realize this is the result of a community contribution but we should be careful about what we commit to our codebase as this can have broader impact. 

I believe there are other ways to tackle this problem so bookmarking this for a future refactor effort.